### ISSUE TITLE
Collaborator Unique Values 

### ISSUE THREAD

**Issue by WMAprogramme** [newdesigncongress/wma-hk-web#10]
Can the name of collaborator be the same in Chinese Locales? 

Some collaborators only have english name and would like to have the same attribute for Chinese Locales

**Comment by shibacomputer** [comment]
Yes they can. You can put English into any of the Chinese Locale fields in the CMS and Chinese into any English field.
The ONLY place where Chinese cannot be used is in site slugs.

**Comment by ClauLung** [comment]
We can't save it if the ENG name and CH name are the same. one example here is Pierfrancesco Celada
<img width="952" alt="Screenshot 2023-10-11 at 4 44 45 AM" src="https://github.com/newdesigncongress/wma-hk-web/assets/95728978/3a245bff-ae9e-409a-b21c-4b5d227f9b17">
